cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/ApplyLogic.java

```diff
@@ -60,18 +60,20 @@ public void applyApplications(@Nonnull Map<String, ApplicationBean> desiredAppli
         DiffLogic diffLogic = new DiffLogic();
         Log.info("Comparing the applications...");
         DiffResult wrappedDiff = diffLogic.createDiffResult(liveApplicationsConfig, desiredApplicationsConfig);
+        System.out.println(diffLogic.createDiffOutput(liveApplicationsConfig, desiredApplicationsConfig));
```

Comment:
Was for debugging purposes, should be removed again

Reply:
removed